UtilityTest.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+from Utility import insert_into_User, retrieve_from_user
+import sqlite3
+
+
+class TestUserFunctions(unittest.TestCase):
+
+    def test_insert_into_User_successful(self):
```

Comment:
This test case fails. Error reads "Table "user" does not exist". This could be due to a multitude of reasons most likely because you are incorrectly accessing the database 

Go back and ensure you are referencing it correctly